backend/app/api/v1/endpoints/task.py

```diff
@@ -70,11 +71,11 @@ async def read_tasks(
         f"{SystemMessages.LOG_ATTEMPT_FETCH_TASKS.format(query=query, skip=skip, limit=limit)}"
     )
     try:
-        admin = token_data.role == "admin"
+        admin = token_data.role == SystemMessages.ADMIN
```

Comment:
Can we return the value from this function `Depends(get_token_data)`? 

Please rename the variable as is_admin if it is containing bool values.

Reply:
bhaia, actually it is containing the string of the user role, extracted from the token's payload